### ISSUE TITLE
type bool, but is initialized to an incompatible value of type int when using string.contains 

### ISSUE THREAD

**Issue by Noobware1** [ethanblake4/dart_eval#131]
version -  0.7.0
trying to do ```bool isFillerEpisode = 'test'.contains("ssl-item-filler");```

```
CompileError: Type mismatch: variable "isFillerEpisode is specified as type bool, but is initialized to an incompatible value of type int at unknown (file package:extenstions_test/main.dart)
#0      Compiler.compileSources (package:dart_eval/src/eval/compiler/compiler.dart:497:7)
#1      Compiler.compile (package:dart_eval/src/eval/compiler/compiler.dart:164:12)
#2      compilerEval (package:extenstions_test/extenstions_test.dart:80:28)
<asynchronous suspension>
#3      main (package:extenstions_test/extenstions_test.dart:179:20)
<asynchronous suspension>
```

**Comment by ethanblake4** [comment]
Pushed a commit fixing this to v0.7 branch

**Comment by ethanblake4** [comment]
Fixed in v0.7.0
